### ISSUE TITLE
Search Results - Sidecar Images - Alt Text

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#517]
SC 1.1.1 A – Alternative Text (Informative Images) – Image associated with each search result (10) are
missing alternative text
